src/game/server/hl2/npc_zombine.h

```diff
@@ -0,0 +1,135 @@
+#ifndef NPC_ZOMBINE_H
+#define NPC_ZOMBINE_H
+#pragma once
+
+#include "npc_BaseZombie.h"
+
+enum
+{	
+	SQUAD_SLOT_ZOMBINE_SPRINT1 = LAST_SHARED_SQUADSLOT,
+	SQUAD_SLOT_ZOMBINE_SPRINT2,
+};
+
+#define MIN_SPRINT_TIME 3.5f
```

Comment:
These defines can go back to the cpp file (anything that doesn't have "zombine" in its name shouldn't be in this header.)